doc/source/examples/networktheory/CPW media example.ipynb

```diff
@@ -0,0 +1,269 @@
+    "\n",
+    "# plot them all\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "plt.suptitle('Raw measurements')\n",
+    "plt.subplot(2, 2, 1)\n",
+    "TL100.plot_s_db(0, 0)\n",
+    "TL200.plot_s_db(0, 0)\n",
+    "TL100.plot_s_db(1, 1)\n",
+    "TL200.plot_s_db(1, 1)\n",
+    "plt.subplot(2, 2, 2)\n",
+    "TL100.plot_s_deg(0, 0)\n",
+    "TL200.plot_s_deg(0, 0)\n",
+    "TL100.plot_s_deg(1, 1)\n",
+    "TL200.plot_s_deg(1, 1)\n",
+    "plt.subplot(2, 2, 3)\n",
+    "TL100.plot_s_db(1, 0)\n",
+    "TL200.plot_s_db(1, 0)\n",
+    "TL100.plot_s_db(0, 1)\n",
+    "TL200.plot_s_db(0, 1)\n",
+    "plt.subplot(2, 2, 4)\n",
+    "TL100.plot_s_deg(1, 0)\n",
+    "TL200.plot_s_deg(1, 0)\n",
+    "TL100.plot_s_deg(0, 1)\n",
+    "TL200.plot_s_deg(0, 1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Removing connectors effects\n",
+    "\n",
+    "The SMA connectors cause a discontinuity in the signal path. The waves have to go trough a coaxial to planar transition with a complex three dimensionnal geometry in the launch region.\n",
+    "\n",
+    "By splitting the 100 mm line into two halves and deembed them from the 200 mm line, the connector and 50 mm of line will be removed on both sides. This will remove the effects caused by the connectors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# deembedding using IEEEP370 impedance corrected 2xthru method\n",
+    "dm = IEEEP370_SE_NZC_2xThru(dummy_2xthru = TL100, name = '2xthru')\n",
+    "side1 = dm.s_side1\n",
+    "side1.name = 'side1'\n",
+    "side2 = dm.s_side2\n",
+    "side2.name = 'side2'\n",
+    "d_dut = dm.deembed(TL200)\n",
+    "d_dut.name = 'd_dut'\n",
+    "\n",
+    "# plot them all\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "plt.suptitle('Connector and launch model')\n",
+    "plt.subplot(2, 2, 1)\n",
+    "side1.plot_s_db(0, 0)\n",
+    "side2.plot_s_db(1, 1)\n",
+    "plt.subplot(2, 2, 2)\n",
+    "side1.plot_s_deg(0, 0)\n",
+    "side2.plot_s_deg(1, 1)\n",
+    "plt.subplot(2, 2, 3)\n",
+    "side1.plot_s_db(1, 0)\n",
+    "side2.plot_s_db(0, 1)\n",
+    "plt.subplot(2, 2, 4)\n",
+    "side1.plot_s_deg(1, 0)\n",
+    "side2.plot_s_deg(0, 1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Comparison of deembedded dut with CPW media results\n",
+    "We now compare our deembedded measurements results with simulated results from skrf CPW media.\n",
+    "The ep_r and tanD value are taken from \"Correlating microstripline model to measurement\" notebook in the same directory.\n",
+    "\n",
+    "The insertion loss agreement is very good both in phase and magnitude.\n",
+    "\n",
+    "The return loss is more disapointing. The overall magnitude seems correct but this is the only agreement to be seen. Return losses below -20 dB are tricky to measure in general because the signal become small."
```

Comment:
I don't think the return loss is disappointing, it's quite close to the model. Considering that connectors were removed from the measurement and -20 dB is already very good matching it's ok result.

Reply:
You are right, I will rephrase that with a positive meaning.